en/micro-integrator/docs/setup/deployment/k8s-pipeline/adding-test-cases.md

```diff
@@ -0,0 +1,107 @@
+# Adding test cases
```

Comment:
This is not specific to this page but relevant to all the pages here. We seem to dive right into the "how to setup" part without any introduction. As a result, we are missing the high-level story of these docs. Someone who lands on one of these pages cannot get a quick idea of what they are trying to achieve here. Shall we add an introduction paragraph or two to each of these topics to say what we are trying to do and why we are trying to do it?

Reply:
This is done